rbenv/rbenv.nu

```diff
@@ -0,0 +1,34 @@
+# rbenv
+export env PATH { $env.PATH | split row (char esep) | prepend [$"($env.HOME)/.rbenv/bin" $"($env.HOME)/.rbenv/shims"] }
+export env RBENV_SHELL { "nu" }
+export env RBENV_VERSION { "" }
+export env RBENV_VERSION_OLD { "" }
+
+export def-env rbenv [
+	command?: string@'nu-complete rbenv',
+	...args
+] {
+    let new-env = if $command in ["rehash", "shell"] {
```

Comment:
The upcoming nushell doesn't support variable format like `a-b`, you can rename it to `a_b`